omero_figure/templates/figure/index.html

```diff
@@ -698,8 +698,14 @@ <h4 class="modal-title" id="addImagesLabel">Labels from Key-Value Pairs</h4>
                 </div>
                 <form class="labelsFromMapAnnsForm" role="form">
                     <div class="modal-body">
-                        <p>Choose Key
-                            <small class="text-muted">(shows numbers of images)</small>:</p>
+                        <div>                                                    
+                            <input type="radio" id="all-keys" name="kvpChoice" value="all-keys" />
+                            <label for="all-keys">All Keys </label>
+              
+                            <input type="radio" id="single-key" name="kvpChoice" value="single-key" checked />
+                            <label for="one-key">Choose Key <small class="text-muted">(shows numbers of images)</small>:</label>
```

Comment:
The `for="one-key` and `id="single-key"` should match, so that clicking the label also selects the radio button